packages/kotti-ui/source/kotti-field-select/components/GenericSelectField.vue

```diff
@@ -11,20 +11,13 @@
 				:helpTextSlot="helpTextSlot"
 			>
 				<div class="kt-field-select__input-and-tags">
-					<div
+					<KtTag
 						v-for="option in visibleSelectedTags"
 						:key="option.value"
-						class="kt-field-select__tag"
-					>
-						<div class="kt-field-select__tag-text" v-text="option.label" />
-						<div
-							v-if="!(field.isDisabled || Boolean(option.isDisabled))"
-							class="kt-field-select__tag-icon"
-							@click.stop.prevent="removeTag(option.value)"
-						>
-							<i class="yoco" v-text="Yoco.Icon.CLOSE" />
-						</div>
-					</div>
+						isClosable
```

Comment:
to remember: 
check how the `isDisabled` is implemented

because it probably needs to be controlled from here and exposed throough the api of KtTag